js/mixpanel.js

```diff
@@ -0,0 +1,6 @@
+if (window.location.hostname != "localhost") {
```

Comment:
Let's make this explicit for "intercity.io" so we're certain it only gets initialized on production. Otherwise :+1: on this solution